src/quo/foundations/colors.cljs

```diff
@@ -293,6 +304,7 @@
                     60 warning-60}}
          customization
          networks
+         short-name-map
```

Comment:
hmm so `short-name-map` is a bit vague. Should reflect that's for networks too. Maybe `networks-short-name` ?

Reply:
DOne